rubocop-rspec.gemspec

```diff
@@ -37,7 +37,7 @@ Gem::Specification.new do |spec|
   }
 
   spec.add_runtime_dependency 'rubocop', '~> 0.87'
-  spec.add_runtime_dependency 'rubocop-ast', '~> 0.7.1'
+  spec.add_runtime_dependency 'rubocop-ast', '~> 0.7'
```

Comment:
There was a [specific reason to use 0.7.1 as opposed to 0.7.0](https://github.com/rubocop-hq/rubocop-rspec/pull/1036#issuecomment-701258609). But I guess `~> 0.7` is good either way, and `bundler` will install 0.7.1+ unless 0.7.0 is already installed.

I hope this explicit dependency specification statement will go away when RuboCop and `rubocop-ast` hit 1.0, and we hit 2.0.

Reply:
After giving it another thought, let's write it as `>= 0.7.1`, since 0.7.1 released [this](https://github.com/rubocop-hq/rubocop-ast/pull/127) fix for JRuby.